### PR TITLE
Appveyor status issue

### LINKED ISSUE #373: Update Appveyor Badge
🐞 **Problem**
The appveyor badge turns red whenever a build in any PR is failing. We don't want appveyor badge to reflect any PR's status, but it should report only `master` branch's status.

What I mean is this : 

🎯 **Goal**

Update the badge in `README.rst`

💡 **Possible solutions**

Take a look at [shields](https://shields.io/)

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by vineetg3**
Added /branch/master on appveyor code to reflect only master branch status

Fixes #373

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -42,7 +42,7 @@
    :target: https://codecov.io/github/einsteinpy/einsteinpy?branch=master
 
 .. |appveyor| image:: https://img.shields.io/appveyor/ci/shreyasbapat/einsteinpy.svg?logo=appveyor&style=flat-square
-   :target: https://ci.appveyor.com/project/shreyasbapat/einsteinpy
+   :target: https://ci.appveyor.com/project/shreyasbapat/einsteinpy/master/branch
```
Comment (ritzvik) [inline README.rst]: No change was required here. The change was required in `.. |appveyor| image::` line .

Kindly see `AppVveyor branch` on [this](https://shields.io/category/build) link and update the image url.

Comment (vineetg3) [reply README.rst]: Did the same.

File: README.rst [hunk 2]
```diff
@@ -41,8 +41,8 @@
 .. |codecov| image:: https://img.shields.io/codecov/c/github/einsteinpy/einsteinpy.svg?style=flat-square
    :target: https://codecov.io/github/einsteinpy/einsteinpy?branch=master
 
-.. |appveyor| image:: https://img.shields.io/appveyor/ci/shreyasbapat/einsteinpy.svg?logo=appveyor&style=flat-square
-   :target: https://ci.appveyor.com/project/shreyasbapat/einsteinpy
+.. |appveyor| image:: https://img.shields.io/appveyor/ci/shreyasbapat/einsteinpy/master
```
Comment (ritzvik) [inline README.rst]:
```suggestion
.. |appveyor| image:: https://img.shields.io/appveyor/ci/shreyasbapat/einsteinpy/master?logo=appveyor&style=flat-square
```

File: README.rst [hunk 3]
```diff
@@ -41,8 +41,8 @@
...
+   :target: https://ci.appveyor.com/project/shreyasbapat/einsteinpy/master/branch
```
Comment (ritzvik) [inline README.rst]:
```suggestion
   :target: https://ci.appveyor.com/project/shreyasbapat/einsteinpy/branch/master
```

**Comment by ritzvik** [comment]
I guess apply the suggestions, then it's done.

**Comment by vineetg3** [comment]
> I guess apply the suggestions, then it's done.

done the same. Please check and thank you!

**Comment by ritzvik** [comment]
@vineetg3 Keep up the good work :)
Merging 🥇

**Pull request merged**